test/pre-test-scripts.ts

```diff
@@ -48,7 +51,12 @@ async function runMigrations() {
 }
 
 const seedDb = async () => {
-  //
+  new ThirdParty();
```

Comment:
Why instanciate an empty class here? Can we remove it? I don't think it's doing anything.

Reply:
Oh my mistake, I deleted that line